### PR TITLE
Messaging Screen #12

### PR DESCRIPTION
**Pull request by dwij2812**
Completed the tasks assigned and tested on all possible platforms available.

### CODE REVIEW

File: views/inbox.ejs [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+                    <img class="icon-top" src="../media/council_logos/EU.JPG" width="120">
+                </a>
+            </div>
+            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu" style="color: #ffffff;">
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+            </a>
+        </div>
+        <div id="navbarBasicExample" class="navbar-menu">
```
Comment (the-lost-explorer) [inline views/inbox.ejs]: Can we change this to dash casing?
```suggestion
        <div id="navbar-basic-example" class="navbar-menu">
```

File: views/messages.ejs [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                    <img class="icon-top" src="../media/council_logos/EU.JPG" width="120">
+                </a>
+            </div>
+            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+            </a>
+        </div>
+        <div id="navbarMenu" class="navbar-menu">
```
Comment (the-lost-explorer) [inline views/messages.ejs]: Have you used the id or class anywhere? Or is it redundant. If so one of them should be removed.

File: views/messages.ejs [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+            <a class="button gold-button is-bold">
+                <span class="compose">
+                    <strong>
+                        REPLY
+                    </strong>
+                </span>
+            </a>
+        </div>
+    </div>
+    <script async type="text/javascript" src="../public/js/bulma.js"></script>
```
Comment (the-lost-explorer) [inline views/messages.ejs]: Let's add the javascript includes in `<head>` tag itself. With proper comments.

File: views/partials/message_screen(pg2).html [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+                            </div>
+                            </div>
+                          </div>
+                        </div>
+                    </div>
+                    </div>
+            </div>
+        </div>
+    </div>
+    <script async type="text/javascript" src="../public/js/bulma.js"></script>
```
Comment (the-lost-explorer) [inline views/partials/message_screen(pg2).html]: Add it to header.

File: views/partials/message_screen.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+                    <a href="#" class="item"><span class="icon"><i class="fa fa-envelope-o"></i></span><span class="name">Sent Mail</span></a>
+                </div>
+            </div>
+        </aside>
+        <div class="column is-10">
+            <div class="columns">
+            </div>
+        </div>
+    </div>
+    <script async type="text/javascript" src="../public/js/bulma.js"></script>
```
Comment (the-lost-explorer) [inline views/partials/message_screen.html]: Add it to header.

File: views/partials/messages.html [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+            <a class="button gold-button is-bold">
+                <span class="compose">
+                    <strong>
+                        REPLY
+                    </strong>
+                </span>
+            </a>
+        </div>
+    </div>
+    <script async type="text/javascript" src="../public/js/bulma.js"></script>
```
Comment (the-lost-explorer) [inline views/partials/messages.html]: Add it to the header.

File: views/messages.ejs (hunk 2 above)
Comment (dwij2812) [reply views/messages.ejs]: It is used for the Nurger menu in nav bar

Comment (the-lost-explorer) [reply views/messages.ejs]: I think the class might be a bulma class?

Comment (dwij2812) [reply views/messages.ejs]: Nah the id is used by the JS action to change the burger menu icons etc.

File: views/messages.ejs (hunk 3 above)
Comment (dwij2812) [reply views/messages.ejs]: Sorry, that was included by mistake, its fixed up now (I removed it)

File: views/messages.ejs (hunk 2 above)
Comment (the-lost-explorer) [reply views/messages.ejs]: Okay then!

File: views/partials/message_screen(pg2).html (hunk 4 above)
Comment (dwij2812) [reply views/partials/message_screen(pg2).html]: Resolved

File: views/partials/message_screen.html (hunk 5 above)
Comment (dwij2812) [reply views/partials/message_screen.html]: Resolved

File: views/partials/messages.html (hunk 6 above)
Comment (dwij2812) [reply views/partials/messages.html]: Resolved

**Pull request merged**